### PR TITLE
tpm2_import: support specifying parent key via context file

### LINKED ISSUE #1023: tpm2_import specify handle via context file.
The parent handle is required to be persistent in the tpm2_import tool. @joshuagl has been working on rectifying options, so wait until that work is done, so we know what option to choose for a parent handle.

See:
https://github.com/tpm2-software/tpm2-tools/pull/982/files

### PR DESCRIPTION
**Pull request by joshuagl**
Update to support specifying the parent key as either a handle *or* a context file. Change the option long and short forms:
-H -> -C
--parent-key-handle -> --parent-key-context

Fixes #1023

### CODE REVIEW

File: man/tpm2_import.1.md [hunk 1]
```diff
@@ -21,30 +21,31 @@ OPTIONS
 
 These options control the key importation process:
 
-  * `-G`, **--import-key-alg**=_ALGORITHM_:
+  * **-G**, **--import-key-alg**=_ALGORITHM_:
     The hash algorithm to use. Algorithms should follow the
```
Comment (williamcroberts) [inline man/tpm2_import.1.md]: Not form your patch, but I think this description is wrong. -G should be the Algorithm for the key iteself, -g is the name algorithm which is a hashing algorithm.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ## Changelog
 ### next
+  * tpm2_import: support specifying parent key with a context file,
+    --parent-key-handle/-H becomes --parent-key-context/-C
```
Comment (williamcroberts) [inline CHANGELOG.md]: We should probably do `--parent-key`, since it can be either a context or a handle.

Comment (joshuagl) [reply CHANGELOG.md]: agreed, will change.

File: man/tpm2_import.1.md (hunk 1 above)
Comment (joshuagl) [reply man/tpm2_import.1.md]: had to dig through code and docs to confirm, but I believe you're right. I'll fix and resubmit.

**Comment by joshuagl** [comment]
Review comments addressed and added a bonus patch to tidy up some manual page formatting that I noticed whilst fixing the description for `tpm2_import`'s **-G** option.

**Pull request merged**